### DISCUSSION TITLE
How best to include the effects of corporate income taxes?

### DISCUSSION THREAD

**Discussion by martinholmer** [PSLmodels/Tax-Calculator#2825] [General]
**Focus**
This issue has been opened to serve as a place to discuss how best to include the effects on individuals of changes in aggregate corporate income tax revenue.

**Background**
This topic has been discussed in a series of earlier issues and pull requests:
- closed issue #223 first introduced this topic (Apr 2015)
- closed pull request #286 was the first suggested implementation (Jun 2015)
- closed issue #315 contains a revised discussion of this topic (Jul 2015)
- merged pull request #316 added documentation on the revised approach (Jul 2015), which is available [here](https://github.com/open-source-economics/Tax-Calculator/blob/master/read-the-docs/proposed_features/DistributingCorpIncTax.rst).

Then after two years of no discussion,
- open pull request #1482 contained a second suggested implementation (Jul 2017)

**Problem**
The problem with all these discussions and proposed implementations is that they were not informed by the economic theory of the effects of changes in the corporate income tax.  All the cited references were to other microsimulation models and how they mechanically represent corporate income taxes.  Those cited references contained little or no discussion of the economic literature on the incidence of corporate income taxes.  Without a clear understanding of the economic literature, there is no basic economic understanding to guide our work.

**Solution**
At the end of the discussion of #1482, there was this exchange:

@martinholmer said (Aug 2017):
> @yuying27 said after providing links to and short summaries of Treasury and JCT papers:
>
>> For the labor income and capital income definition, as well as the weights assigned, different articles vary from each other. So I think in our formula, it’s better to focus on one paper. Otherwise, we will lose the consistency.
>
> Thanks for the links and summaries, but what I'm more interested in is **economic analysis** of the effects of a reform-induced change in corporate income tax (CIT) revenue. The Treasury and JCT papers, like the TPC paper, focus on the **distribution mechanics**. Eventually, we will have to address issues of mechanics, but before we do that we need to understand what the most recent thinking is on the economic analysis, both theoretical and empirical.
>
> The most recent review of theoretical and empirical issues I could find, is [this paper](http://piketty.pse.ens.fr/files/Clausing2012.pdf):
K. Clausing, "In Search of Corporate Tax Incidence", Tax Law Review (2012). This paper also includes new empirical research.
>
> Do you want to begin there? Or can you suggest a better article to begin with?

To which @yuying27 replied (Aug 2017):
>> Do you want to begin there? Or can you suggest a better article to begin with?
>
> Definitely. I will start with this article.

**Discussion**
We invite anyone with an interest in this topic to join in the discussion of how best to include the effects on individuals of a change in aggregate corporate income tax revenue.  In order to have everybody's comments in one place, we're asking people to add their comments to this issue.

@MattHJensen @jdebacker @rickecon @Amy-Xu @andersonfrailey @hdoupe @codykallen

**Comment by MattHJensen** [comment]
Here is a review paper that I wrote with Aparna Mathur a few years ago on this topic. 

http://www.aei.org/wp-content/uploads/2011/06/Tax-Notes-Mathur-Jensen-June-2011.pdf

**Comment by MattHJensen** [comment]
Today there is an [article](https://www.wsj.com/articles/who-ultimately-pays-for-corporate-taxes-the-answer-may-color-the-republican-overhaul-1502184603) on this topic in the Wall Street Journal.

**Comment by jdebacker** [comment]
[Here's](https://www.cbpp.org/research/federal-tax/corporate-tax-cuts-skew-to-shareholders-and-ceos-not-workers-as-administration) a survey of the empirical literature by the CBPP.

**Comment by martinholmer** [comment]
@jdebacker,  Thanks for providing in #1515 the link to the CBPP review of the literature.

**Comment by martinholmer** [comment]
Now that I have read the three papers (and one newspaper article) cited in issue #1515, I have a couple of comments about how best to model the effects of a change in corporate income tax (CIT) revenues.  Notice that the change in CIT revenues is an input parameter for this modeling because nobody is suggesting that we should build a CIT model that has corporations as filing units and the provisions of CIT law as policy parameters.

The **first comment** is that I'm not very familiar with this area of economics, so I may be operating under some misunderstandings.  If that is so, please do not hesitate to set me straight about whatever I'm not understanding about the economic literature on the CIT.

The **second comment** is that seems to be little disagreement about the mechanisms by which corporations might shift some of the CIT burden to labor.  The economic reasoning is classic dynamic general-equilibrium analysis (as all three papers make clear).  The main mechanism is that a reduction in CIT will increase domestic investment, and over time, raise the domestic capital-labor ratio, and hence, raise average domestic wages.  How much individual income taxes will change for a filing unit depends on the individual income tax rate schedule as well as on how much that filing unit's capital income increases and on how much that filing unit's labor income increases.  However, there is considerable disagreement over the relative magnitude of capital income increases and labor income increases caused by a reduction in CIT revenues.  But that is not so much of a problem for us because all the uncertain parameters in the dynamic general-equilibrium analysis can be made into model parameters that are under the control of model users.

The **third comment** is that given the type of analysis involved in analyzing CIT incidence, the obvious place for us to add CIT analysis is in the OG-USA dynamic general-equilibrium model.  I know that is not how other tax microsimulation models work, but their choices were forced by the fact that they had no dynamic general-equilibrium model available at the time (or, in some cases, still do not have a dynamic general-equilibrium model as part of their income tax analysis capabilities).

Given my third comment, I don't understand why pull request #1482 is trying to add CIT analysis (which is inherently a dynamic general-equilibrium analysis) to a static microsimulation model.

@MattHJensen @feenberg @jdebacker @rickecon
@Amy-Xu @andersonfrailey @hdoupe @codykallen @yuying27

**Comment by MattHJensen** [comment]
I agree that modeling changes to the corporate income tax, including the process that might lead those changes to impact labor productivity and therefore wages, could be a useful project for OG-USA when that team has the bandwidth and if they are interested. 

It also seems that dynamic models, generally, or econometric models -- are the best tools for estimating the impact on corporate income taxes on labor. 

But I don't think that PR #1482 is inconsistent with that view. PR #1482 does not attempt to estimate the impact of corporate income tax changes on various forms of capital and labor -- instead, it will allow tax-calculator users to apply estimates of those impacts to distribute the burden. 

Those estimates will almost certainly be derived from dynamic models. 

The way I think about this -- and I believe the way TPC, CBO, JCT, and OTA think about this (all of whom do have general equilibrium models, by the way) -- is that policymakers and other users are interested in seeing a comprehensive distributional table with as many tax programs as possible. Currently and in the foreseeable future, our best foundation for developing that comprehensive table is Tax-Calculator+TaxData.

**Comment by martinholmer** [comment]
@MattHJensen said in the #1515 discussion of CIT modeling approaches:
> Currently and in the foreseeable future [until OG-USA is more developed], our best foundation for developing that comprehensive table [that includes the incidence of the CIT] is Tax-Calculator+TaxData.

OK, but this kind of CIT analysis is **not static** in any meaning of the term.  So, it seems to me that the CIT distribution logic should be one of Tax-Calculator's behavioral responses.  This would mean that all the parameters related to CIT analysis would be in `behavior.json` and all the code that implements the CIT analysis would be in `behavior.py`.  And after that is done, TaxBrain can offer CIT analysis as another "Partial Equilibrium Simulation" option.  Does that make sense?

**Comment by martinholmer** [comment]
@MattHJensen said in #1515:
> I believe the way TPC, CBO, JCT, and OTA think about this (all of whom do have general equilibrium models, by the way) ...

Yes, they do now.  But my point was when they first "distributed CIT to individuals" none of them had dynamic general-equilibrium models.  So, the only way to go about this taks was to combine it with static tax analysis, which is not very logical.

@MattHJensen also said:
> [the approach in #1482] will allow tax-calculator users to apply estimates of those impacts to distribute the burden.

I don't see it that way; I see #1482 as logically flawed.  The change in CIT revenues assumed in the CIT analysis causes changes in capital income and (perhaps) labor income.  The individual income tax implications of those changes in **incomes** depends on the magnitude and distribution of those income changes and also depends on the individual income tax rates for those types of income.  So, simply dividing up the assumed aggregate change in CIT revenue, and distributing the shares of the aggregate CIT revenue change across individuals, does not lead to correct estimates of the changes in individual income taxes that would be produced by the changes in incomes generated by a change in aggregate CIT revenues.

Do you agree?  Or am I missing something here?

**Comment by MattHJensen** [comment]
@martinholmer said: 

> So, simply dividing up the assumed aggregate change in CIT revenue, and distributing the shares of the aggregate CIT revenue change across individuals, does not lead to correct estimates of the changes in individual income taxes that would be produced by the changes in incomes generated by a change in aggregate CIT revenues.

> Do you agree? Or am I missing something here?

I agree. I had not thought about the need to capture the interaction between the CIT and the income tax. @yuying27, what do you think?

**Comment by yuying27** [comment]
> @martinholmer said:

>> So, simply dividing up the assumed aggregate change in CIT revenue, and distributing the shares of the aggregate CIT revenue change across individuals, does not lead to correct estimates of the changes in individual income taxes that would be produced by the changes in incomes generated by a change in aggregate CIT revenues.
Do you agree? Or am I missing something here?

>I agree. I had not thought about the need to capture the interaction between the CIT and the income tax. @yuying27, what do you think?

I agree. I did not consider the interaction neither. I was mainly following the logic of #223 and did not think about dynamic or static problem.
Then it is necessary to do lots of modification of codes.

**Comment by martinholmer** [comment]
**Summary**

Summarizing the #1515 discussion of how best to include the effects of corporate income taxes (CIT) in Tax-Calculator, there seems to be consensus on a few central points:

- Both @MattHJensen and @yuying27 agree with the following point:
   > I see #1482 as logically flawed. The change in CIT revenues assumed in the CIT analysis causes changes in capital income and (perhaps) labor income. The individual income tax implications of those changes in incomes depends on the timing and on the magnitude and distribution of those income changes, and also depends on the individual income tax rates for those types of income.
   >
   > So, simply dividing up the assumed aggregate change in CIT revenue, and distributing the shares of the aggregate CIT revenue change across individuals, does not lead to correct estimates of the changes in individual income taxes that would be produced by the changes in incomes generated by a change in aggregate CIT revenues.

- However it is done, it would seem that estimating the CIT-induced changes in capital and labor income is not part of static tax analysis.  It involves some kind of dynamic analysis, perhaps reduced-form partial-equilibrium analysis (via an expanded Behavior class) or perhaps structural dynamic analysis (using a growth model whose results feedback to affect individual capital and labor income in Tax-Calculator).  Or perhaps others can suggest an alternative approach.

**Implications**

Many thanks to @yuying27 for all the hard work on pull request #1482, which focused our attention on theoretical questions and without which we would have not made as much conceptual progress as we have.
But we seem to be in agreement that #1482 is not the best approach to modeling the impact on individual taxes of changes in aggregate CIT revenues.  Given this conclusion, #1482 is being closed.  But it will always remain part of the history of the Tax-Calculator repository, providing help in future work on CIT effects.

**Comment by MattHJensen** [comment]
@martinholmer, thanks very much for the extensive summary, and thanks again to @yuying27 for your excellent work on this project. @yuying27, if you have availability again later this fall or in the future, you are very welcome to continue your contributions to this project!

**Comment by martinholmer** [comment]
Tax-Calculator issue #1515 --- entitled "How best to include the effects of corporate income taxes?" --- is in the process of being resolved by @codykallen's work in the BRC repository, which will in the coming months become part of the Policy Simulation Library's collection of USA tax models.  I'm going to leave this issue open until that process has been completed.  

I plan to begin soon helping @codykallen make the changes to BRC that will make it PSL compliant.  I now have a good sense, after creating the Behavioral-Responses repo, about how to help @codykallen organize his code so that eventually we will be able to create a conda package for BRC.  That will allow users to write Python scripts in which that package and the Tax-Calculator taxcalc package work together to estimate the aggregate and distributional effects of a corporate income tax reform.

@MattHJensen @andersonfrailey @hdoupe
